packages/types/src/settings.ts

```diff
@@ -66,6 +68,8 @@ export const SettingsSchema = z.object({
       baseUrl: z.union([z.literal(""), z.string().trim().url()]).optional(),
     })
     .optional(),
+  ollamaModel: z.string().optional(),
+  ollamaBaseUrl: z.string().optional(),
```

Comment:
note: if you do `z.string().optional().default("http://")`, this gets converted into a type where the property is no longer considered optional, which breaks an assumption about the Settings type, i.e., that `Settings` is a partial object 

providing a default value marks the property as non-optional